WcaOnRails/spec/controllers/persons_controller_spec.rb

```diff
@@ -43,6 +43,14 @@
         expect(json['total']).to eq 2
         expect(json['rows'].count).to eq 2
       end
+
+      it "works well when parts of the name are given" do
+        xhr :get, :index, search: "Law Jenn"
```

Comment:
Two people is a nice sample! I'd like to see the tests with 

`"umber"` - does middle of name work?
`"Jen Ben" - returns two users or none?
`"CUMB" - case sensitive?
`"er%"` - are we escaping properly?
`"16law"` - do partial and wrongly cased IDs work?